### ISSUE TITLE
Support for 128 bit floating point numbers?

### ISSUE THREAD

**Issue by v0dro** [SciRuby/nmatrix#320]
I just read [here](http://en.wikipedia.org/wiki/Quadruple-precision_floating-point_format#Computer-language_support) that GCC supports 128 floating point numbers on intel hardware as data type `__float128`.

Would it be possible to implement this in nmatrix? Maybe only for intel x86 machines?

**Comment by mohawkjohn** [comment]
Yes — the problem being that ATLAS doesn't support quads. We could do this, but it would need to be part of eliminating the unnecessary rational types. Why only for Intel x86 machines, out of curiosity?

**Comment by mohawkjohn** [comment]
I'm not putting a milestone on this enhancement because I think it needs some more design work before it can really be considered achievable.
